configs.py

```diff
@@ -31,6 +31,8 @@ def pip_install(module_name):
                            command=["python3.7", "-m", "universum.analyzers.pylint", "--python-version=3.7",
                                     "--rcfile=pylintrc", "--result-file=${CODE_REPORT_FILE}",
                                     "--files", "*.py", "universum/", "tests/"]),
+                      dict(name="Run static type checker", code_report=True,
+                           command=["python3.7", "-m", "mypy", "universum/main.py", "--ignore-missing-imports"]),
```

Comment:
'glob2', 'sh', and 'urllib3' need stubs to remove the import errors
or we can just wait until their maintainers add type annotations themselves